elastalert/alerts.py

```diff
@@ -1076,6 +1077,15 @@ def get_aggregation_summary_text(self, matches):
             text = u'```\n{0}```\n'.format(text)
         return text
 
+    def populate_fields(self, matches):
+        slack_alert_fields = self.rule['slack_alert_fields']
+        alert_fields = []
+        if 'slack_alert_fields' in self.rule:
+            for arg in slack_alert_fields:
+                arg['value'] = lookup_es_key(matches[0], arg['value'])
```

Comment:
Add this before line 1085 to fix the issue reporter about subsequent alerts failing.
```
arg = copy.copy(arg)
```